demo/sass/demo.scss

```diff
@@ -52,6 +52,10 @@ article {
 	@extend #{$prefix}.clear;
 }
 
+.hljs {
+	border: 1px solid #ccc;
```

Comment:
you can load one of the color defined in lucca-ui, see this [doc](https://github.com/LuccaSA/lucca-ui/wiki/Theming-%282.0%29#retrieving-theme-variables). it should look something like that
```sass
@import "../../scss/lucca-ui.global.scss";
$grey: $primaryColor: luiPalette(grey, color);
.hljs {
  border: 1px solid $grey;
  ...
}
```
ask @bpaugam if you need help